### DISCUSSION TITLE
Does kohya suffer from this gradient accumulation bug?

### DISCUSSION THREAD

**Discussion by ilazarte** [kohya-ss/sd-scripts#1716] [Q&A] [answered]
Hello, this blog post describes a supposedly universal gradient accumulation bug, and I was wondering if someone was knowledgeable enough to know whether kohya training scripts (including this repo) suffers from this or it doesn't apply: https://unsloth.ai/blog/gradient

**Comment by kohya-ss** [comment, marked as answer]
From my understanding, this issue only occurs with cross entropy loss. Currently, in this repository, we use MSE loss, so this issue should not occur.

**Reply by ilazarte** [reply]
thanks for reviewing!
